### PR TITLE
Unit Test for Model Objects  for classes in org.eclipse.pass.object.model

### PR DESCRIPTION
**Pull request by jrmartino**
We adapted the old test classes from pass-model in the pass-java-client project. Since we no longer need the JSON <-> object (de)serialization, these test methods were omitted. The new tests reflect changes made to the model, especially adapted for use with our new storage regime.

### CODE REVIEW

**Review by markpatton — COMMENTED** [review]
We have an incoming pr that does some test setup with a newer version of junit. Let's get that in and then update this to the newer junit

**Comment by jrmartino** [comment]
made changes to the unit tests to use jupiter, looks clean

File: pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/Identifier.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
+/*
```
Comment (markpatton) [inline pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/Identifier.java:1]: Should not be needed.

File: pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/IdentifierTests.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
+/*
```
Comment (markpatton) [inline pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/IdentifierTests.java:1]: Not needed.

**Review by markpatton — COMMENTED** [review]
I don't think the Identifier and Identifier test classes are needed. Otherwise looks good.

File: pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/Identifier.java (hunk 1 above)
Comment (jrmartino) [reply pass-core-object-service/src/test/java/org/eclipse/pass/object/model/support/Identifier.java]: right you are, have pushed deletion of these files

**Review by markpatton — APPROVED** [review]
Looks good now.

**Pull request merged**